engine/execution/ingestion/engine_test.go

```diff
@@ -875,3 +875,40 @@ func TestLoadingUnexecutedBlocks(t *testing.T) {
 			pending)
 	})
 }
+
+func TestChunkfiyEvents(t *testing.T) {
+	// generate events
+	events := make([]flow.Event, 0)
+	for j := 0; j < 10; j++ {
+		events[j] = unittest.EventFixture(flow.EventAccountCreated, uint32(j), uint32(j), unittest.IdentifierFixture())
+	}
```

Comment:
```suggestion
	var events []flow.Event
	for j := 0; j < 10; j++ {
		events = append(events, unittest.EventFixture(flow.EventAccountCreated, uint32(j), uint32(j), unittest.IdentifierFixture()))
	}
```

I think the way it is now will cause index out of bound error

Reply:
fixed at cfc8bcf